src/main/java/pl/edu/agh/firecell/engine/algorithm/FirePropagator.java

```diff
@@ -98,6 +108,18 @@ private int computeBurningTimeWood(Cell oldCell, double newTemperature, int curr
         return newBurningTime;
     }
 
+    private boolean hasNeighbourLongOnFire(State oldState, Vector3i cellIndex){
+        int requiredTime = 10;
```

Comment:
MAAAGIC NUUUMBEEERS

Reply:
My question about it was above in comments, please answer

Reply:
It should be static final field inside FirePropagator, since this number is connected with fire propagation and only propagator uses it <3